app/utils/mail.server.ts

```diff
@@ -47,11 +47,16 @@ export const sendEmail = async (
 }
 
 export const sendTeamInviteEmail = async (email: string, app: string) => {
-  return await sendEmail(email, "Your invite to POKT Portal", EmailTemplates.TeamInvite, {
-    app: app,
-    // TODO: Change to correct invite link
-    invite_link: "https://www.portal.pokt.network/account",
-  })
+  return await sendEmail(
+    email,
+    "Your invite to Grove Portal",
+    EmailTemplates.TeamInvite,
+    {
+      app: app,
+      // TODO: Change to correct invite link
```

Comment:
"https://www.portal.grove.city/user/invited-apps" ?

Reply:
yeah that makes sense